lib/spack/spack/cmd/install.py

```diff
@@ -378,6 +381,9 @@ def get_tests(specs):
     if args.deprecated:
         spack.config.set('config:deprecated', True, scope='command_line')
 
+    if args.terminal_title:
+        spack.config.set('config:set_terminal_title', True, scope='command_line')
```

Comment:
There should be an equivalent option to turn off titles if the config default is turned on

Reply:
I added a `--no-terminal-title` option to turn it off explicitly. Not sure if this is what you meant.